### ISSUE TITLE
Сборка?

### ISSUE THREAD

**Issue by dehix228** [sadsalat/Orignal-Homigrad#22]
Можете запостить эту сборку хомиграда из этого видео? https://youtu.be/xHNoODdi7tY?si=ylVPYxNmZEIDwJ59 если стремно переходить по ссылке то видео называеться либо в окно либо пулю хомиград я бы хотел сборку из этого видео

**Comment by uzelezz123** [comment]
Плыв

**Comment by MrPointlezz** [comment]
Плыв

**Comment by magmen3** [comment]
Плыв

**Comment by Pisanw** [comment]
Согласен, лучше запостить новую сборку, хотя может данная сборка не для сандбокса, почему Салат не выложит новую сборку?
Потому-что он ленивая фем бойская попа

**Comment by magmen3** [comment]
какая нахер новая сборка идиоты это опенград на видео

**Comment by magmen3** [comment]
и не скидывайте сюда больше говноэдиты по калограду с треком инстасамки на фоне

**Comment by magmen3** [comment]
вы тупые потому что все

**Comment by uzelezz123** [comment]
Я согласен с магменом

**Comment by Solt17** [comment]
Мне кажется настоящие тупицы юзелесс и магмен. Вы не сливаете сборку нового хомиграда потому что вам насрать на тех людей которые хотят поиграть со сборкой нового хомиграда. Вы и поэтому отнекиваетесь якобы это опенград.

**Comment by uzelezz123** [comment]
> Мне кажется настоящие тупицы юзелесс и магмен. Вы не сливаете сборку нового хомиграда потому что вам насрать на тех людей которые хотят поиграть со сборкой нового хомиграда. Вы и поэтому отнекиваетесь якобы это опенград. Это касается и садсалата.

Ладно придется выложить новый хомиград

Вот ссылка на видео там будет ссылка на новый гитхаб https://youtu.be/dQw4w9WgXcQ?si=dLDxBJh_KPizcrgJ

**Comment by Kernantin** [comment]
молимся буде чтоб они выложили ноый хомиград или какую то новую сборку

**Comment by magmen3** [comment]
Надейся дальше
